app/Http/Requests/EInvoice/Peppol/CreateRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2024. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+namespace App\Http\Requests\EInvoice\Peppol;
+
+use App\Models\Country;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        /**
+         * @var \App\Models\User
+         */
+        $user = auth()->user();
+
+        return $user->account->isPaid();
+    }
+
+    /**
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'party_name' => ['required', 'string'],
+            'line1' => ['required', 'string'],
+            'line2' => ['nullable', 'string'],
+            'city' => ['required', 'string'],
+            'country' => ['required', 'string'],
+            'zip' => ['required', 'string'],
+            'county' => ['required', 'string'],
+        ];
+    }
+
+    public function prepareForValidation(): void
+    {
+        $country = Country::findOrFail($this->country);
```

Comment:
this fails _hard_ instead, pass in null, that way the validation can advise the user of the location of the issue?